### ISSUE TITLE
The app's player page does not have a Chinese accessibility tab.

### ISSUE THREAD

**Issue by anmi2004** [TeamNewPipe/NewPipe#10196]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

It is hoped that the player page of the application and the playback control bar below will be added with a Chinese accessibility label.

### Why do you want this feature?

I am blind and cannot operate the player page properly when using the app.

### Additional information

Accessibility related content is listed in Google's Accessibility Development Guide.

**Comment by TobiGr** [comment]
Please test the APK provided in #10199
